src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -277,6 +286,26 @@ public function get_context( ?string $store_namespace = null ): array {
 			? $context[ $store_namespace ]
 			: array();
 	}
+	/**
```

Comment:
Missing space here

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/7474